src/main/java/com/example/GymProject/GymProjectApplication.java

```diff
@@ -1,13 +1,19 @@
 package com.example.GymProject;
 
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
 
-@SpringBootApplication
+import com.example.GymProject.config.AppConfig;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan(basePackages = "com.example")
 public class GymProjectApplication {
 
 	public static void main(String[] args) {
-		SpringApplication.run(GymProjectApplication.class, args);
+		ApplicationContext context = new AnnotationConfigApplicationContext(AppConfig.class);
```

Comment:
Just `new AnnotationConfigApplicationContext(AppConfig.class);`without assigning the resulting object to any var. You will not use this `context` variable, right? Do not hesitate do get rid of it.